src/Questions/Recaptcha/index.js

```diff
@@ -0,0 +1,28 @@
+import ReCAPTCHA from 'react-google-recaptcha'
+
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx } from 'theme-ui'
+
+const QuestionRecaptcha = ({ recaptchaRef, formDataValues, onSubmitForm, question }) => {
+  const onReCAPTCHAChange = async (captchaCode) => {
+    if (!captchaCode) {
+      return
+    }
+
+    recaptchaRef.current.reset()
```

Comment:
I thing React refs are nullable. Might need to add optional chaining here.
```suggestion
    recaptchaRef.current?.reset()
```